src/osa/nightsummary/database.py

```diff
@@ -60,47 +52,47 @@ def query(obs_id: int, property_name: str):
     if not isinstance(obs_id, int):
         obs_id = int(obs_id)
 
-    caco_client = MongoClient(CACO_DB)
-    tcu_client = MongoClient(TCU_DB)
-
-    with caco_client, tcu_client:
-        run_info = caco_client["CACO"]["RUN_INFORMATION"]
-        run = run_info.find_one({"run_number": obs_id})
+    try:
+        tcu_client = MongoClient(TCU_DB, serverSelectionTimeoutMS=3000)
+        db = tcu_client["lst1_obs_summary"]
+        camera_col = db["camera"]
 
-        try:
-            start = datetime.fromisoformat(run["start_time"].replace("Z", ""))
-            end = datetime.fromisoformat(run["stop_time"].replace("Z", ""))
-        except TypeError:
-            return None
+        run_summary = camera_col.find_one({"run_number": obs_id})
```

Comment:
this is a very minor comment, but maybe is better to rename this as run_info or something similar, since at the beginning I got confused with this name thinking it was referring to the general run summary of the night